server/lib/models/account.go

```diff
@@ -8,12 +8,13 @@ import (
 	"crypto/sha256"
 	"encoding/base64"
 
-	proto "../../proto"
+	proto "github.com/umegaya/channer/server/proto"
```

Comment:
__[gometalinter]__
* [gotype] could not import github.com/umegaya/channer/server/proto (can't find import: github.com/umegaya/channer/server/proto)